internal/transport/transport_test.go

```diff
@@ -461,11 +461,21 @@ func setUpWithOptions(t *testing.T, port int, sc *ServerConfig, ht hType, copts
 }
 
 func setUpWithNoPingServer(t *testing.T, copts ConnectOptions, connCh chan net.Conn) (*http2Client, func()) {
+	return setUpWithNoPingServerWithOptions(t, copts, connCh, nil)
+}
+
+// setUpWithNoPingServerWithOptions sets up a server that can be toggled between
+// responding to ping acks and not responding based on the writes to the respModeCh
+// channel.
+//
+// * If `true` is written to respModeCh, the server will respond to ping acks. (default) 
```

Comment:
vets are failing because of the trailing whitespace. please take a look @tobotg 
```suggestion
// * If `true` is written to respModeCh, the server will respond to ping acks. (default)
```